packages/client/modules/meeting/components/PokerTemplateListOrgRoot.tsx

```diff
@@ -21,16 +13,14 @@ interface Props {
 
 const PokerTemplateListOrgRoot = (props: Props) => {
   const {isActive, teamId} = props
-  const atmosphere = useAtmosphere()
+  const queryRef = useQueryLoaderNow<PokerTemplateListOrgQuery>(pokerTemplateListOrgQuery, {teamId})
```

Comment:
-1 we don't want to trigger a network request until `isActive` is true. easiest fix is to probably move the `isActive` logic to the parent.

TEST
- [ ] open dev tools -> network -> websocket & monitor traffic
- [ ] open the poker meeting settings
- [ ] this query should _not_ get fetched until the organization tab is clicked

Reply:
good point! This is the difference between component-based usage and React hook, and I've updated it!